Perpendicular flap: Document a stronger coupling condition

Fujikawas opened the pull request:
With this change the coupling becomes stronger for the perpendicular flap case. It will require implicit coupling AND acceleration methods to converge. With $\rho=42$, it requires only implicit coupling, no acceleration is necessary. With current $\rho=3000$, both explicit and implicit coupling could lead to convergence.
With $\rho=1$, the result from `fluid-openfoam`-`solid-dealii`  looks like:
![image](https://github.com/precice/tutorials/assets/94080048/c2051a0b-c13f-4f53-b6d6-75daf0ec97a8)
With $\rho=42$, the result from `fluid-openfoam`-`solid-dealii`  looks like:
![image](https://github.com/precice/tutorials/assets/94080048/387042fc-f013-4dfa-9418-d319341b8d6b)

Checklist:

- [ ] I added a summary of any user-facing changes (compared to the last release) in the `changelog-entries/<PRnumber>.md`.
- [ ] I will remember to squash-and-merge, providing a useful summary of the changes of this PR.

Fujikawas committed:
make coupling stronger in perpendicular flap

uekerman commented:
Worth considering on top: We could make the fluid inflow time-dependent. Then, also the $\rho=3000$ scenario could look interesting again.

MakisH commented:
@Fujikawas please add in a comment the new figures you got in the context of https://github.com/MakisH/ofw19-training

MakisH commented:
This has high educational value, as it demonstrates a case in which implicit coupling is important.
To not trigger a cascade of side changes that require a lot of effort, let's convert this to a documentation note, encouraging the user to try a different density and observe the effect. (@Fujikawas)

Fujikawas committed:
Merge branch 'develop' into strong-coupled-flap

Fujikawas committed:
improve documentation

Fujikawas committed:
reverse the setting change

Fujikawas committed:
formatting

MakisH committed:
Move section above disclaimer

MakisH committed:
Fix typos/language

MakisH committed:
Remove intermediate argumentation points

Implicit coupling without acceleration is not well-motivated and rather confuses.

MakisH reviewed (approved):
I directly fixed a few issues and removed the intermediate step of ρ=42 and no acceleration, as that I think is more confusing than helps.

Current rendering:

<img width="859" height="200" alt="Screenshot from 2025-07-31 21-26-50" src="https://github.com/user-attachments/assets/686b99fc-ab3f-4272-a35c-a00b42c3d1d5" />
